aws/table_aws_iam_access_advisor.go

```diff
@@ -32,8 +33,9 @@ func tableAwsIamAccessAdvisor(_ context.Context) *plugin.Table {
 		Description:      "AWS IAM Access Advisor",
 		DefaultTransform: transform.FromGo(),
 		List: &plugin.ListConfig{
-			KeyColumns: plugin.SingleColumn("principal_arn"),
-			Hydrate:    listAccessAdvisor,
+			KeyColumns:        plugin.SingleColumn("principal_arn"),
+			ShouldIgnoreError: isNotFoundError([]string{"InvalidParameter"}),
```

Comment:
It looks like the code below avoids making the list call if the account id is not in this account.  If thats the case, should we ignore the error?